RESULTS.md

```diff
@@ -0,0 +1,37 @@
+# Documenting results
+
+This is a very important part of your benchmarking process. Proper documentation allows you to easily decide between algorithms that are best fit for your use case, in the future. 
+
+Please use Markdown files to write the benchmark document.   
+
+You can use this template to give all the relevant results.
+
+### <center>Algorithm Name</center>
+
+#### Introduction
+Contains a short description about your algorithm, and what the algorithm is trying to achieve. Should also contain pseudo-code if possible (especially if the algorithm is not a well known algorithm).
+
+#### Parameters
+This section should contain the list of all parameters (and a short description for non trivial parameters), constant or otherwise. The values of constant parameters should be provided. Also, the range of other parameters that are going to vary in your test bench should also be provided.
+
+#### Test Cases
+You should provide the complete list of test cases, providing links to databases/video files/ROS bag files that have been used in your test bench.
+
+#### Results
+This section details your result. It should preferably be presented in tabulated form, for easy understanding. Your table should include the test case name, values of each variable used, and the values of the benchmarking variables obtained in the test. Quick graphs can also be given.
+
+#### Footer
+This section will contain other assorted details, like the assumptions used, and your conclusions. This can also contain other relevant links, like links to research papers on the particular algorithm, github repositories, etc. 
+
+## Example Report
+For a good example you can see the benchmarking report of ORB_feature_detector algorithm, in this repository. Some general comments regarding the example report given :-
+
+1) If you can easily find the pseudo-code for your algorithm or are well-versed in LaTeX and able to write one easily and quickly, you must provide the pseudo-code. This is so that their is less downtime in trying to understand the actual code, should someone want to implement this algorithm in a different language/package. In the example, it hasn't been provided as the pseudo-code isn't available easily and this function is already implemented in the well-known OpenCV library (so anyone who wants to implement it elsewhere can directly use the OpenCV functions).
```

Comment:
I doubt that whether spelling mistakes should be pointed out or not.If it should be ,then change "This is so that 'their' " to "This is so that 'there' ".

Reply:
Of course, they should be :)